src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/DatabaseBackupTests.cs

```diff
@@ -125,5 +125,27 @@ public void TestDatabaseGeoBackupPolicy()
                 RunPowerShellTest("Test-DatabaseGeoBackupPolicy");
             }
         }
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestNewDatabaseRestorePoint()
+        {
+            // TODO Rewrite SQL backup tests to be recordable
```

Comment:
Are these recordable? If not, please have instructions on the tests describing how to record them.

Reply:
They are. The comments are general to look over all these tests added by Jared. For future refactoring.